runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -313,6 +313,12 @@ public interface FlinkPipelineOptions
 
   void setFlinkConfDir(String confDir);
 
+  @Description("Set the maximum size of input split when data is read from a filesystem.")
+  @Default.Long(128 * 1024 * 1024)
```

Comment:
We could let the default be the old behavior, and only set this for pipelines that are affected. If you have good evidence that this is generally helpful and rarely hurtful providing a good default makes sense.